lms/services/canvas_api/factory.py

```diff
@@ -19,15 +18,9 @@ def canvas_api_client_factory(_context, request):
     canvas_host = urlparse(ai_getter.lms_url()).netloc
     basic_client = BasicClient(canvas_host)
 
-    token_store = TokenStore(
-        consumer_key=request.lti_user.oauth_consumer_key,
-        user_id=request.lti_user.user_id,
-        db=request.db,
-    )
```

Comment:
The factories for services like `CanvasAPIClient` and in future `BlackboardAPIClient` don't need to know how to create a `TokenStore` or what `TokenStore`'s dependencies are. They just find it with `request.find_service(name="token_store")`. This is nice because we don't end up with multiple places in the code where a `TokenStore` is created and its dependencies injected. `token_store_service_factory()` is the only place in the code where a `TokenStore` is created